modules/core/classes/GalleryUnknownItem.class

```diff
@@ -40,6 +40,9 @@ GalleryCoreApi::requireOnce('modules/core/classes/GalleryDataItem.class');
  */
 class GalleryUnknownItem extends GalleryDataItem {
 
+    /* deprecated dynamic properties in php 8.2 */
+    var $parent;
```

Comment:
I don't think this is needed

Reply:
Without the variable declaration unit tests 562, 1149 and 1612 fails with the deprecation warning from php 8.2